lib/style/pipes.ex

```diff
@@ -225,6 +225,8 @@ defmodule Styler.Style.Pipes do
   defp valid_pipe_start?({{:., _, _}, _, _}), do: false
   # variable
   defp valid_pipe_start?({variable, _, nil}) when is_atom(variable), do: true
+  # macro with single block argument
+  defp valid_pipe_start?({fun, _, [[{{:__block__, _, [:do]}, {:__block__, [], []}}]]}) when is_atom(fun), do: true
```

Comment:
I have no idea if this is the right thing to do. I am very familiar with the Elixir AST, but when it is combined with the Zipper, I am less confident in what I am seeing.

Reply:
at a glance, the final `[]` in the second `{:__block__` looks overspecifying - i think you're saying "fun/macro with an _empty_ do block" rather than just "with a do block"